### DISCUSSION TITLE
how do I export a file for then use the import button?

### DISCUSSION THREAD

**Discussion by ofernandofilo** [victornpb/undiscord#325] [1. Questions & Answers] [answered]
I didn't saw / found any documentation about it.

thx!

**also**: the pinned button DON'T work for me... I can live without it, but it is much better using it! [tested on channel only, not in guilds]

cheers!

**Comment by victornpb** [comment, marked as answer]
https://github.com/victornpb/undiscord/wiki/Deleting-all-messages-automatically
